src/main/java/org/gridsuite/loadflow/server/computation/utils/ContextUtils.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (c) 2023, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package org.gridsuite.loadflow.server.computation.utils;
+
+import com.powsybl.commons.PowsyblException;
+import org.springframework.messaging.MessageHeaders;
+
+/**
+ * @author Thang PHAM <quyet-thang.pham at rte-france.com>
+ */
+public final class ContextUtils {
```

Comment:
I'm not sure this is the right name 
It's not relative to a context but a message
`MessageUtils`

Reply:
Agreed. Changed. Is this ok for @thangqp the maker of this class ?